### PR TITLE
docs: simplify neovim glimpse.menu shortcut

### PR DESCRIPTION
**Pull request by jasonkuhrt**
Hey @MarcoIeni! I found this to work, and it seems simpler.

### CODE REVIEW

File: website/docs/install.md [hunk 1]
```diff
@@ -97,26 +97,13 @@ Add the following:
 
 ### [VSCode Neovim](https://marketplace.visualstudio.com/items?itemName=asvetliakov.vscode-neovim) users
 
-Add the following to your `init.vim`:
```
Comment (jasonkuhrt) [inline website/docs/install.md:100]: Is there any reason to still keep these instructions? Maybe some users want to keep configuration in their Neovim configuration files?

Comment (MarcoIeni) [reply website/docs/install.md:100]: Yes, exactly.
I would like to keep both the approaches documented. Can you edit the PR to put your approach before the existing one?
You can describe the existing approach as "alternative".

**Comment by MarcoIeni** [comment]
Thanks!

**Comment by jasonkuhrt** [comment]
Thanks @MarcoIeni!

**Pull request merged**